relay-server/src/utils/rate_limits.rs

```diff
@@ -124,7 +124,7 @@ fn infer_event_category(item: &Item) -> Option<DataCategory> {
         ItemType::Statsd => None,
         ItemType::MetricBuckets => None,
         ItemType::FormData => None,
-        ItemType::UserReport => None,
+        ItemType::UserReport => Some(DataCategory::UserReportV2),
```

Comment:
I don't think this is correct, is it?

I think user report v1 are not counted as events.

Reply:
> What do you mean by "not counted as events"?

What the description of the function describes. Is the User v1 report considered an 'event'? If not you shouldn't be changing this function.